### PR TITLE
Incorporate workflow building t-network solidity-contracts

### PR DESCRIPTION
**Pull request by michalinacienciala**
We're adding a new module (`threshold-network/solidity-contracts`) to
the CI flow configuration. As the module should be built before
`tbtc-v2` and `coverage-pools` modules and will be changed more often
than `tbtc`, it's placed between `tbtc` and `tbtc-v2` in the CI flow.
Apart from changing the `config.json file, we're also updating actions
which rely on this file.

Ref:
https://github.com/threshold-network/solidity-contracts/pull/76

TODO after merging:
- [ ] update the v1 tag

### CODE REVIEW

**Comment by michalinacienciala** [comment]
Now that I think about it, we probably need to do some additional, more complex changes in the actions, because the workflow we're adding to the CI flow belongs to a different organization than the `ci` project (`threshold-network` vs `keep-network`) and we will not be able to authorize triggering of the workflow in `threshold-network` using `keep-network`'s `GITHUB_TOKEN`.
@nkuba, do you think would it be possible to add a `threshold-network`'s token as a secret in `keep-network` and modify code of our actions to support differentiation between the tokens for `threshold-network` and `keep-network`?

**Comment by michalinacienciala** [comment]
I discussed the issue with @Kuba. We substitute `GITHUB_TOKEN` with `CI_GITHUB_TOKEN`, which is a token for `thesis-heimdall` user. We should be able to trigger `threshold-network` workflows from `keep-core/ci` repo if `thesis-heimdall` user gets added as a member of `threshold-network` organization - this needs to be done by @pdyraga.

File: actions/notify-workflow-completed/dist/ceda2c0264ac98da5d36.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "defaultModule": "github.com/keep-network/keep-core/solidity-v1",
+    "defaultModule": "github.com/keep-network/keep-core/solidity",
```
Comment (nkuba) [inline actions/notify-workflow-completed/dist/ceda2c0264ac98da5d36.json:2]: Those generated files look incorrect, aren't they out of sync with the actual source code?

**Review by nkuba — CHANGES_REQUESTED** [review]

**Comment by michalinacienciala** [comment]
@nkuba, I looked at your change (e609f35b65f84cb04cb068eb5f268a52aa18a738), looks ok to me. Once you review my changes, we should be ready to merge. After the merge I will update the tag for the actions

**Review by nkuba — APPROVED** [review]

**Pull request merged**